core/storage/feedback/gae_models.py

```diff
@@ -287,13 +404,29 @@ def create(cls, exploration_id, thread_id, author_id, exploration_version,
 
     @classmethod
     def _get_instance_id(cls, exploration_id, thread_id):
+        """Returns ID of instance the exploration and thread belongs to.
+
+         Args:
+             exploration_id: str. ID of exploration.
+             thread_id: str. ID of thread.
+
+         Returns:
+             str. ID of instance.
+        """
         return '.'.join([exploration_id, thread_id])
 
     @classmethod
     def get_by_exploration_and_thread_id(cls, exploration_id, thread_id):
         """Gets a suggestion by the corresponding exploration and thread id's.
 
-        Returns None if it doesn't match anything."""
+        Args:
+            exploration_id: str. ID of exploration.
+            thread_id: str. ID of thread.
+
+        Returns:
+            str. Suggestion related to given exploration and thread id, or none
```

Comment:
In the not-None case, what is the type of the return value? Either look at callsites to find out, or try and simulate the behaviour using the dev server and log/print the type so that it shows up in the server logs.